response_operations_ui/templates/sign_in.html

```diff
@@ -93,7 +93,7 @@ <h2>Sign in</h2>
                     "name": "username",
```

Comment:
This is still the username (email and username are now the same value) and why it hasn't been changed. It's purely to make things easier for the user to change it to Email address